src/core/agent/modules/signify/signifyApi.ts

```diff
@@ -284,18 +285,25 @@ export class SignifyApi {
     aid: Aid,
     otherAids: Pick<Aid, "state">[],
     name: string,
-    delegate?: Aid
+    delegate?: Aid,
+    threshold?: number
   ): Promise<{
     op: any;
     icpResult: EventResult;
     name: string;
   }> {
+    if (!threshold) {
+      threshold = otherAids.length + 1;
+    }
+    if (threshold < 1 || threshold > otherAids.length + 1) {
+      throw new Error(SignifyApi.INVALID_THRESHOLD);
+    }
```

Comment:
Test for threshold too small and test for threshold too high. And in the existing tests check that the threshold of the created identifier is what we passed (or at least that the signify mock was called with this